BtSendData/app/src/main/java/com/example/myapplication/operatePage.java

```diff
@@ -24,6 +24,9 @@
     private ImageButton stopbutton;
     private ImageButton returnButton;
 
+    public ImageButton getDownwardArrow(){
```

Comment:
might not need this. just made a getter because i couldnt reach the private varable at first